Makefile

```diff
@@ -30,7 +30,7 @@ uninstall: kustomize
 
 # Install Helm into a cluster
 helm-install:
-	helm install falcon-helm falcon.cid=$(CID) ./helm-charts/falcon-sensor
+	helm install -n falcon-system --create-namespace falcon-helm falcon.cid=$(CID) ./helm-charts/falcon-sensor
```

Comment:
If we're creating namespaces, should the naming be more generic, e.g. ``crowdstrike-falcon``?

Reply:
We could do `crowdstrike-falcon-system` but having `$NAME-system` is kinda a thing.... e.g. `kube-system` and others.... In fact, when you create operators, this is the appropriate naming.

Reply:
thanks for sharing context on this!